tree/ntuple/v7/src/RColumnElement.hxx

```diff
@@ -409,6 +409,64 @@ public:
    }
 }; // class RColumnElementDeltaSplitLE
 
+/// Reading of unsplit integer columns to boolean
+template <typename CppIntT>
+class RColumnElementBoolAsUnsplitInt : public RColumnElementBase {
+protected:
+   explicit RColumnElementBoolAsUnsplitInt(std::size_t size, std::size_t bitsOnStorage)
+      : RColumnElementBase(size, bitsOnStorage)
+   {
+   }
+
+public:
+   static constexpr bool kIsMappable = false;
+
+   // No support for writing an integer into a boolean
```

Comment:
The unpack code assigns an array of 'int' value into an array of `bools` which can be read as "writing an integer into a boolean".  Is the comment incorrect or am I missing something?

Reply:
Write in the sense of "write to disk" here. I clarified the comment.